test/update-version-readme-test.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * update-version-readme-test.js
+ *
+ * A unit test for update-version.js
+ *
+ * Copies the fixtures to the temp folder and runs update-version
```

Comment:
Missing `.` at end of sentence.

Reply:
like "A unit test for update-version.js." ?